lte/gateway/c/core/oai/tasks/nas/emm/EmmCommon.h

```diff
@@ -135,4 +136,5 @@ struct emm_common_data_s* emm_common_data_context_get(
 void create_new_attach_info(
     emm_context_t* emm_context_p, mme_ue_s1ap_id_t mme_ue_s1ap_id,
     struct emm_attach_request_ies_s* ies, bool is_mm_ctx_new);
+partial_list_t* emm_verify_orig_tai(const tai_t orig_tai);
```

Comment:
can you please include a comment on the function logic, i.e., what is the expected return based on the input?